### DISCUSSION TITLE
Does the rotation of the bin matter?

### DISCUSSION THREAD

**Discussion by ageekhere** [CorsixTH/CorsixTH#2409] [Research Room]
Does the rotation of the bin matter? or can patient access it from any direction?

**Comment by lewri** [comment]
Its orientation does not matter, the patient does not visibly use it but its tile placement has an effect:
https://github.com/CorsixTH/CorsixTH/blob/87858c2d5a4060a6039ca4110fb24403d17d4123/CorsixTH/Lua/entities/humanoids/patient.lua#L798-L802

**Reply by ageekhere** [reply]
When it says "litter bin catches 8 radius" how many tiles is the bin radius?

**Reply by Alberth289346** [reply]
Path finder does 4 directions search, so rectangular shape of 8 around the bin.

I guess it attracts a lot of attention :D
8 may be somewhat too much.

**Reply by ageekhere** [reply]
I wonder if influence radius could be shown when items are selected.

**Reply by ghost** [reply]
Would be nice to see the effect radius (and happiness + value) when moving bins and radiators around a room before placing them.This kind of info is so hidden.

**Comment by Alberth289346** [comment]
Would be too much information. A large part of the fun of a game is finding out what things work. You'd destroy a large part of that if you give all those statistics away

**Reply by MandrakeHorse** [reply]
Unlike radiators, which have a significant impact on your staff and patients' happiness and therefore needs to be managed, it wouldn't be that useful to know the exact area of influence for each bin. Most players will simply place a bin in every room and then another one in the corridors whenever they spot litter.

**Comment by ageekhere** [comment]
To be clear on the area affect on bins, how many tiles away does a patient need to be in order to use a bin? Does it need to be in their walk path? e.g if you have a room that is 5x5 does the bin placement matter?

Also i think it could be cool to add a little animation when a patient uses a bin. Like a little banana icon that floats up from the patient.
